reactNativeExample/.buckconfig

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Can we put the example app in `/example/reactnative`?

Reply:
moved to /example/reactnative